simpledotorg/simple-server
# Add diabetes downloads

**Story card:** [sc-7794](https://app.shortcut.com/simpledotorg/story/7794/monthly-district-data-download)
**Story card:** [sc-7796](https://app.shortcut.com/simpledotorg/story/7796/monthly-facility-data-download)

## Because

We need to report diabetes monthly district data and monthly facility data in CSV format

## This addresses

Make diabetes monthly district data and monthly facility data downloadable in CSV format

## Test instructions

- Make sure the Flipper flag named diabetes_management_reports is enabled
- Go to Report > Diabetes > Downloads(DM)
- You should be able to see the reports listed in the menu
- Click it to download and open to see the reports
